timrlink.net.CLI/DatabaseContext.cs

```diff
@@ -74,9 +74,16 @@ public class ProjectTime
         public DateTime LastModifiedTime { get; set; }
         public string Task { get; set; }
         public string Description { get; set; }
-
         public bool Billable { get; set; }
         public DateTimeOffset? Deleted { get; set; }
+        public Guid? UserUUID { get; set; }
+        public string? UserExternalId { get; set; }
+        public string? UserEmployeeNr { get; set; }
+        public DateTimeOffset? StartTimeOffset { get; set; }
```

Comment:
Why is this duplicating `StartTime` now? We only need one. `StartTime` should be migrated from `DateTime` to `DateTimeOffset`